### DISCUSSION TITLE
How is the segment table encoded?

### DISCUSSION THREAD

**Discussion by asyncth** [capnproto/capnproto#2142] [Q&A] [answered]
What are the fields of the segment table and what do they represent? Couldn't find any information about it on the website.

**Comment by kentonv** [comment, marked as answer]
This is covered under "Serialization Over a Stream":

https://capnproto.org/encoding.html#serialization-over-a-stream

The bullet list there describes the segment table. Not sure why I didn't literally use the words "segment table" when writing the doc.

**Reply by asyncth** [reply]
Thanks, not sure how did I miss it despite looking through this page multiple times.
